apps/landing/components/Plans/PlanPeriod.tsx

```diff
@@ -145,7 +145,7 @@ export const PlanPeriod: React.FC<{
         if (!open) select();
       }}
       className={cn(
-        "border border-natural-100 bg-natural-50 rounded-2xl p-4",
+        "border border-natural-100 bg-[#FCFCFC] rounded-2xl p-4",
```

Comment:
Let's move this color into a variable. It should be added first to `tailwind.css` then to `tailwind.config.js`